app/components/workbench/DiffView.tsx

```
@@ -0,0 +1,755 @@
+  }) => {
+    const bgColor = diffLineStyles[type];
+
+    const renderContent = () => {
+      if (type === 'unchanged' || !block.charChanges) {
+        const highlightedCode = highlighter
+          ? highlighter
+              .codeToHtml(content, { lang: language, theme: theme === 'dark' ? 'github-dark' : 'github-light' })
+              .replace(/<\/?pre[^>]*>/g, '')
+              .replace(/<\/?code[^>]*>/g, '')
+          : content;
+        return <span dangerouslySetInnerHTML={{ __html: highlightedCode }} />;
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)